### DISCUSSION TITLE
api代理相关问题

### DISCUSSION THREAD

**Discussion by ShangChien** [ChatGPTNextWeb/NextChat#349] [General]
使用这个一键部署在vercel上，还需要用户使用代理来确保自己的api不被检测为异常吗？

**Comment by machinefriendly** [comment]
api被检测为异常指的是什么？被谁检测？什么是异常？

**Reply by ShangChien** [reply]
就是openai认为你不在他们的用户区域，将你的账号禁用

**Comment by youbaowangsheng** [comment]
请问这个有好的解决方案吗？

**Reply by ShangChien** [reply]
研究后发现其实不用担心这个问题，vercel算是主流云服务器厂商，从其调用API是很正常的操作。
